webapp/src/components/CreateEstatePage/EstateSelect/EstateSelect.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Container } from 'semantic-ui-react'
+
+import ParcelPreview from 'components/ParcelPreview'
+import EstateSelectActions from './EstateSelectActions'
+import Parcel from 'components/Parcel'
+import { t } from 'modules/translation/utils'
+import { isOwner } from 'modules/parcels/utils'
+import { coordsType } from 'components/types'
+
+import './EstateSelect.css'
+
+export default class EstateSelect extends React.PureComponent {
+  static propTypes = {
+    x: PropTypes.number.isRequired,
+    y: PropTypes.number.isRequired,
+    error: PropTypes.string,
+    onCancel: PropTypes.func.isRequired,
+    onContinue: PropTypes.func.isRequired,
+    onChange: PropTypes.func.isRequired,
+    value: PropTypes.arrayOf(coordsType).isRequired
+  }
+
+  isNeighbour = (x, y) => coords => {
```

Comment:
What do you think of moving these methods to something an utils file like `stateUtils.js`?

Reply:
Great idea! Will do after the changes review of this same file :)

Reply:
Added a TODO comment eebd404